Taskfile.yml

```
@@ -9,45 +9,42 @@ tasks:
   testenv:
     desc: 'do docker-compose file for setup test env'
     dir: '{{.USER_WORKING_DIR}}'
     preconditions:
       - test -f infra/docker-compose.yml
     cmds:
       - scripts/setup_test_env.sh
+
   tidy:
     desc: install pkg listed in go.mod
     cmds:
       -  go mod tidy
```

YAMLlint (1.35.1)

[warning] too many spaces after hyphen

(hyphens)